src/main/java/com/example/bookstore/service/CartService.java

```diff
@@ -0,0 +1,28 @@
+package com.example.bookstore.service;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.example.bookstore.dao.CartDetailRepository;
+import com.example.bookstore.dao.CartRepository;
+import com.example.bookstore.entity.Cart;
+import com.example.bookstore.entity.CartDetail;
+
+@Service
+public class CartService {
+	@Autowired
+	private CartRepository cartRepository;
+	@Autowired
+	private CartDetailRepository cartDetailRepository;
+	public List<CartDetail> getCartDetailsByUserId(int userId){
```

Comment:
```suggestion
	public List<CartDetail> getCartInfoByUserId(int userId){
```
cái này mới hợp lý nha, vì điểm chính là em load cart rồi mới lấy cart details của cart nè